src/api/routers/participants/participantsRouter.ts

```diff
@@ -210,7 +210,13 @@ export function createParticipantsRouter() {
 
   participantsRouter.put('/', createParticipant);
 
-  participantsRouter.use('/:participantId', checkParticipantId);
+  participantsRouter.use('/:participantId', hasParticipantAccess);
+
+  participantsRouter.get('/:participantId', async (req: ParticipantRequest, res: Response) => {
```

Comment:
This method sort of looks like it doesn't do anything, but the participant has been enriched by the middleware, so it now contains the fully loaded participant